src/EnergyPlus/DataOutputs.cc

```diff
@@ -173,7 +173,7 @@ bool FindItemInVariableList(EnergyPlusData &state, std::string_view const KeyedV
     // This function looks up a key and variable name value and determines if they are
     // in the list of required variables for a simulation.
 
-    auto const found_variable = state.dataOutput->OutputVariablesForSimulation.find(VariableName);
+    auto const &found_variable = state.dataOutput->OutputVariablesForSimulation.find(VariableName);
```

Comment:
And here. Code compiles and runs with no diffs but I still question whether correct.